docs/guide/writing/style-guides.md

```diff
@@ -2,18 +2,18 @@
 
 A style guide is a set of standards for the writing and design of content, defining the style to be used in communication within a particular organization. Basically, style guides are put together to clarify [the way a group of people talk and write about the things they do](https://www.writethedocs.org/style-guide/). Think of authoring best practices.
 
-If you happen to have a background in academia or journalism, you will probably be familiar the [AP stylebook](https://en.wikipedia.org/wiki/AP_Stylebook) or the [Chicago manual of style](http://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html). Those are great resources for writing in general, particularly for grammar and syntax, but if you're reading this page chances are you are considering style in the context of technical documentation.
+If you happen to have a background in academia or journalism, you will probably be familiar the [AP Stylebook](https://en.wikipedia.org/wiki/AP_Stylebook) or the [Chicago Manual of Style](http://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html). Those are great resources for writing in general, particularly for grammar and syntax, but if you're reading this page chances are you are considering style in the context of technical documentation.
 
-### Technical Documentation
 Style guides help you write a variety of content, such as documenting the methods of an API, or presenting an overview of complex technical concepts, or focusing on how to write particular content like user manuals, release notes, or tutorials.
 
 Guides often help writers focus on the different readers of technical documentation, describing how to adapt content to different reader profiles, like developers, product managers, the general public, and others.
 
 Many focus on the language itself (tone, style, grammar).
 
-Others go beyond the content and discuss the organisation of the documentation, providing best practices on how to manage your content, version control, or publication and delivery strategies. While this is not what we focus on in our style guide, how your documentation is organized, and how easy it is for your readers to find what they are looking for, can be as important as the content itself.
+Others go beyond the content and discuss the organisation of the documentation, providing best practices on how to manage your content, version control, or publication and delivery strategies. While this is not what we focus on in this style guide for style guides, how your documentation is organized and the ease at which your readers can find what they are looking for, can be as important as the content itself.
```

Comment:
ease with which, maybe?

Reply:
I think actually it should be "ease in which".

Reply:
I'm not super convinced, but maybe it is a US/UK thing? Regardless not gonna die on this hill. Merging.